tests/components/component-playground/selected-fixture-and-editor/events/handlers.js

```diff
@@ -60,4 +60,33 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
             .to.equal(JSON.stringify(fixture.state.fixtureContents, null, 2));
     });
   });
+
+  describe('Splitpane orientation', function() {
+    function simulateWindowResize(width, height) {
+      sinon.stub(component.refs.contentFrame, 'getDOMNode').returns({
```

Comment:
why these tests are not here by default? because skidding wrote them https://github.com/skidding/react-component-playground/blob/85aac13d6ef9c65eab4c7f81579d4e597e8f6ec1/tests/components/component-playground/selected-fixture/events/handlers.js#L23-L50